test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -1106,9 +1107,6 @@ test
     await t.expect(await enrollOktaVerifyPage.sameDeviceSetupOnMobileTextExist()).eql(false);
 
     await t.expect(enrollOktaVerifyPage.getAppStoreHref()).eql(sameDeviceOVEnrollmentAndroidLink);
-    if (!userVariables.gen3) {
-      await t.expect(enrollOktaVerifyPage.getOVSetupHref()).eql(sameDeviceOVEnrollmentSetupLink);
-    }
```

Comment:
no need for this assertion since we are checking the page url after clicking the button below